### PR TITLE
Feature: Social login (Facebook)

### PR DESCRIPTION
**Pull request by JsCodder**
some details...

> a
> b

### CODE REVIEW

File: login.js [hunk 1]
```diff
@@ -0,0 +1 @@
+login to Facebook
```
Comment (MrkTheCoder) [inline login.js:1]: Good but capitalize the L in login!

**Review by MrkTheCoder — CHANGES_REQUESTED** [review]
Please make this changes and do another push.

File: login.js (hunk 1 above)
Comment (JsCodder) [reply login.js:1]: Sure I will do!

**Review by MrkTheCoder — APPROVED** [review]
Everything is good. Let's go ahead with merging!

**Pull request merged**